src/app/common-ui/Popup/index.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import styles from './style.scss';
+import PropTypes from 'prop-types';
+
+import CloseButton from '../../../assets/images/close-button.png';
+
+class Popup extends Component {
+  render() {
+    const { closePopup, titleContent, mainContent, contentBtn, linkTo, isSuccess } = this.props;
+    return (
+      <div className={styles.overlay}>
+        <div className={styles.container}>
+          <div className={styles.titleBar}>
+            <h5
+              className={`${styles.titleContent} ${
+                isSuccess === '1' ? styles.titleColorSuccess : styles.titleColorFailed
+              }`}
+            >
+              {titleContent}
+            </h5>
+            {linkTo === '' ? (
+              <div onClick={closePopup}>
+                <img className={styles.closeImg} src={CloseButton} alt="close" />
+              </div>
+            ) : (
+              <Link to={linkTo} onClick={closePopup}>
+                <img className={styles.closeImg} src={CloseButton} alt="close" />
+              </Link>
+            )}
+          </div>
+          <div className={styles.content}>
+            <p>{mainContent}</p>
```

Comment:
Wrap `content` in a `<div>` instead.

`content` might be a React node instead of just a string